src/node_process_events.cc

```diff
@@ -60,18 +60,18 @@ Maybe<bool> ProcessEmitWarningGeneric(Environment* env,
   if (!String::NewFromUtf8(env->isolate(), warning).ToLocal(&args[argc++]))
     return Nothing<bool>();
 
-  if (type != nullptr) {
-    if (!String::NewFromOneByte(env->isolate(),
-                                reinterpret_cast<const uint8_t*>(type))
-             .ToLocal(&args[argc++])) {
-      return Nothing<bool>();
-    }
-    if (code != nullptr &&
-        !String::NewFromOneByte(env->isolate(),
-                                reinterpret_cast<const uint8_t*>(code))
-             .ToLocal(&args[argc++])) {
-      return Nothing<bool>();
-    }
+  if (type != nullptr &&
+      !String::NewFromOneByte(env->isolate(),
+                              reinterpret_cast<const uint8_t*>(type))
+           .ToLocal(&args[argc++])) {
+    return Nothing<bool>();
+  }
+
+  if (code != nullptr &&
```

Comment:
It's not clear if this is breaking / significant or not, but this changes the check here. Previously, `code` would only be checked if `type` was not nullptr. After the chnage `code` is checked independently of `type`.

Reply:
I don't think so as no regression failures were present.  Also when going through the code not sure will there be case when type is null and we do not want to check for nullptr.

Reply:
This is a breaking change since at someplaces we have sent type only and not the code. Probably having the tests to cover the same should be present. Working on that.

Reply:
@jasnell I looked at the checks will only fail when the string if of `k:maxlength` in v8 and it is unlikely for internal deprecations and should not affect the working current or extended.